src/components/modals/TradePreviewModalGP.vue

```diff
@@ -865,6 +942,16 @@ watch(blockNumber, () => {
       >
         {{ $t('approveLidoRelayer') }}
       </BalBtn>
+      <BalBtn
```

Comment:
Agree with Simeon, let's take the opportunity to refactor and use BalActionSteps here, it will take quite a bit of work but if we don't do it as part of this work we might never do it. Also please remove GP from the component name,

Reply:
Good idea, done.